index.src.html

```diff
@@ -318,20 +329,33 @@
     </ul>
   </section>
 
-  <section algorithm="'create a credential' internal method">
-    <dfn for="Credential" method export>\[[Create]](options)</dfn> is called with a
-    {{CredentialCreationOptions}}, and returns either a {{Credential}}, if one can be created, 
-    `null` if no credential was created, or an error if creation fails due to exceptional situations
+  <h5 id="algorithm-create-cred" algorithm>'create a credential' internal method"</h5>
+    <dfn for="Credential" method export>\[[Create]](origin, options)</dfn> is called with
+    with an [=environment settings object/origin=] and a {{CredentialCreationOptions}},
+    and returns either a {{Credential}}, if one can be created,
+    or `null` if no credential was created,
+    or a [=constructCredential=] [=struct=],
```

Comment:
I think you're saying that [[Create]] can 
* create a credential,
* not create one, or
* report an error,

and when creating a credential it can either 
* do it all in parallel (return a `Credential`) or 
* ask to finish up from the main thread (return a `constructCredential`).

Putting the two "success" conditions next to each other would make this clearer, as would saying what they mean.